.drone.yml

```diff
@@ -21,7 +21,7 @@ steps:
     pull: always
     privileged: true
     commands:
-      - zypper -n install helm
+      - curl --cacert /etc/ssl/ca-bundle.pem -sL https://get.helm.sh/helm-v3.13.3-linux-amd64.tar.gz | tar xvzf - -C /usr/local/bin --strip-components=1
```

Comment:
rancher-v2.9 uses its own ssl bundles somewhere else so curl doesn't work without this.

Reply:
This is fine. I don't care whether we leave the `v` flag in the `tar` command or not -- seeing the file get extracted vs. the extra output.